.goreleaser.yml

```diff
@@ -0,0 +1,55 @@
+# This is an goreleaser.yaml file which builds mongodb_exporter releases.
+# Make sure to check the documentation at http://goreleaser.com
+# You can run it locally or with your CI, but you should use `make community-release` or `make snapshot`
+# instead of using `goreleaser` directly as some init steps should be done before it runs.
+dist: bin
+builds:
+  - binary: mongodb_exporter
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - netbsd
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
```

Comment:
Let's keep amd64 only.

Reply:
Same thing: https://github.com/percona/mongodb_exporter/blob/master/.promu.yml#L19
Should we remove those things that we had?

Reply:
Yeah. Otherwise, we just inherit it without knowing why.